.travis.yml

```diff
@@ -1,211 +1,18 @@
-            # This is qgis server address
-            QGIS_SERVER_URL: http://localhost:9000/
-            # This is the location of docker network bridge
-            # So QGIS server can access this address
-            SITEURL: http://localhost:8000/
-            QGIS_SERVER_PORT: 9000
-            ON_TRAVIS: 'True'
-            TEST_RUN_INTEGRATION: 'True'
-            MONITORING_ENABLED: 'False'
-            CELERY_ALWAYS_EAGER: 'True'
-
-branches:
-  only:
-    - master
-    - 2.8.x
-    - 2.6.x
-
-before_install:
-  - mkdir -p $HOME/buildout-cache/{eggs,downloads}
-  - scripts/misc/geoserver_server_setup.sh before_install
-  - scripts/misc/qgis_server_setup.sh before_install
-  - chmod +x scripts/misc/bdd_setup.sh
-  - scripts/misc/bdd_setup.sh before_install
-
-install:
-  - sudo apt-get -qq -y update
-  - sudo apt-get install -y libgdal1h python-gdal gdal-bin spatialite-bin
-  - sudo apt-get install -y python-virtualenv python-imaging python-lxml python-pyproj python-shapely python-nose python-httplib2 python-httplib2 gettext
-  - sudo apt-get install -y python-dev libxml2 libxml2-dev libxslt1-dev zlib1g-dev libjpeg-dev libpq-dev libgdal-dev git default-jdk
-  - sudo add-apt-repository -y ppa:webupd8team/java
-  - sudo apt-get update
-  - sudo apt-get install -y --force-yes oracle-java8-set-default ant maven --no-install-recommends
-  - sudo update-java-alternatives --set java-8-oracle
-  - pip install -r requirements.txt --upgrade
-  - pip install -e . --upgrade
-  - pip install pygdal==`gdal-config --version`
-  - pip install codecov
-
-before_script:
-  # Initialize DB
-  # - sudo apt-get install postgresql-9.6-postgis-2.3 -y -q
-  - psql -U postgres -c "create extension postgis"
-  - chmod +x scripts/misc/create_dbs_travis.sh
-  - scripts/misc/create_dbs_travis.sh before_script
-  # For GeoServer Server Travis steps
-  - scripts/misc/geoserver_server_setup.sh before_script
-  # For QGIS Server Travis steps
-  - scripts/misc/qgis_server_setup.sh before_script
-  - scripts/misc/bdd_setup.sh before_script
-
-  # Start NGINX server
-  - sudo nginx -s stop
-  - sudo nginx -c `pwd`/scripts/misc/nginx_integration.conf
-
-  # Setup GeoNode
-  - paver setup
-
-script:
-  - paver run_tests --coverage --local false
-
-after_script:
-  # For GeoServer Server Travis steps
-  - scripts/misc/geoserver_server_setup.sh after_script
-  # For QGIS Server Travis steps
-  - scripts/misc/qgis_server_setup.sh after_script
-
-  # Stop NGINX server
-  - sudo nginx -c `pwd`/scripts/misc/nginx_integration.conf -s stop
-
-  # Cleanup
-  - paver reset_hard
-
-# after_success:
-  # - codecov
-  # - bash <(curl -s https://codecov.io/bash) -t 2c0e7780-1640-45f0-93a3-e103b057d8c8
-
-notifications:
-  # irc:
-  #   channels:
-  #     - "irc.freenode.org#geonode"
-  #   use_notice: true
-  email:
-    - alessio.fabiani@gmail.com
-  #   - ingenieroariel@gmail.com
-  #   - simone.dalmasso@gmail.com
-  # slack: geonode-sprint:oQylJRkU9feZ8JruGi6czWwe
+language: python
```

Comment:
I don't thinks this change to .travis is coherent with the commit

Reply:
It is not; after I did the pull request I fixed the branch in a couple of minutes, but you have been faster than me :-)